lib/blacklight/xml_parser.rb

```diff
@@ -113,7 +113,8 @@ def self.get_master_parent(pre_data, parents_ids, parent_id)
   # non-metadata file.
   ##
   def self.iterate_files(zipfile)
-    files = zipfile.glob("*/**/**")
+    files = zipfile.entries.select(&:file?)
```

Comment:
I'm not sure this is doing what it's supposed to. When I tested it, the original implementation using `glob` didn't capture any directories. And this new implementation doesn't exclude anything new, it just captures some extra files (such as the .dat files).